src/transformers/models/fuyu/processing_fuyu.py

```diff
@@ -1,32 +1,44 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Image/Text processor class for GIT
+"""
 import re
-from typing import Any, Iterable, List, Optional, Tuple, Union, Dict
+from typing import Dict, List, Optional, Tuple, Union
 
 import numpy as np
 
-from ...image_utils import (
-    ChannelDimension,
-    get_image_size,
-    infer_channel_dimension_format,
-    is_scaled_image,
-    to_numpy_array,
-)
 from ...processing_utils import ProcessorMixin
-from ...tokenization_utils_base import BatchEncoding, PaddingStrategy, TruncationStrategy
-from ...utils import is_torch_available, is_vision_available, logging, requires_backends, is_torch_device, TensorType
+from ...tokenization_utils_base import PaddingStrategy, TruncationStrategy
+from ...utils import TensorType, is_torch_available, is_vision_available, logging, requires_backends
 
 
-if is_torch_available() and is_vision_available():
-    from .image_processing_fuyu import FuyuImageProcessor
+if is_torch_available():
+    from .image_processing_fuyu import FuyuBatchFeature
+
+    if is_vision_available():
+        from .image_processing_fuyu import FuyuImageProcessor
```

Comment:
```suggestion
    from .image_processing_fuyu import FuyuImageProcessor
```

Otherwise I think `import FuyuProcessor` would fail if torchvision is not installed.

Reply:
This helped uncover a bug! The image processor was being reset, overwriting the user's [input here](https://github.com/huggingface/transformers/blob/ae093eef016533a3670561fa9e26addb42d446d1/src/transformers/models/fuyu/processing_fuyu.py#L387). If we get rid of that, then we don't need this import at all

Reply:
Ohhhh, right!